hdr.cpp

```diff
@@ -77,15 +77,15 @@ void ExposureSeries::merge() {
 
 	cout << "Merging " << size() << " exposures .." << endl;
 	#pragma omp parallel for
-	for (size_t y=0; y<height; ++y) {
+	for (int y=0; y<height; ++y) {
 		uint32_t offset = y * width;
```

Comment:
You know what -- never mind. Let's leave all of these variables as ``size_t``. This is really an issue with the compiler which does not fully realize OpenMP 3.0, where iteration variables can be unsigned. In that case, I'd rather have a pragma or CMake compiler flag that shuts up any warnings.